setup_root.md

```diff
@@ -53,6 +55,8 @@ Go inside the new project and modify the selected enviroment variables in the ne
 
 If you wish to know more about enviroment variables you can check [this](env_vars.md).
 
+**Note**: The `data_models_storage_config.json` file defines the connections to the databases that zendro should be aware of.
+
```

Comment:
And currently we have integrated sequelize, neo4j, cassandra, amazon s3, mongodb. For other storage types, user needs to implement the connection part or use generic models. And add a new storage type, a reference doc would be in this PR: https://github.com/Zendro-dev/Zendro-dev.github.io/pull/25